web/README.md

```diff
@@ -16,13 +16,23 @@ PAG Web SDK is built on WebAssembly and WebGL which supports all of the PAG feat
 
 PAG Web SDK consists of two files: `libpag.js` and `libpag.wasm`.
 
-You can use the `locateFile` function to get the path of `libpag.wasm` file. By default, the `libpag.wasm` file is located next to the `libpag.js` file.
-
 ### Browser (Recommend)
 
+Direct `<script>` Include, `libpag` will be registered as a global variable.
+
+For production, we recommend link to a specific version number and build to avoid unexpected breakage from newer versions:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/libpag@4.0.5-release.18/lib/libpag.min.js"></script>
+```
+
+You can browse the files of the NPM package at the public CDN [cdn.jsdelivr.net/npm/libpag/](https://cdn.jsdelivr.net/npm/libpag/) , and you can link to `@latest` to get the latest version.
+
+Other public CDNs that sync with NPM are also available, like [unpkg](https://unpkg.com/libpag@latest/lib/libpag.min.js).
```

Comment:
The PAG library is also available on other public CDNs that sync with NPM,  such as [unpkg]